tueplots/figsizes.py

```diff
@@ -332,8 +362,6 @@ def jmlr2001(
     """JMLR figure size.
 
     Source: https://www.jmlr.org/format/format.html
-
-    The present format is for US letter format.
```

Comment:
Was this information incorrect?

Reply:
My bad, I accidentally deleted this; will undo.